src/ansys/fluent/core/utils/fluent_version.py

```diff
@@ -30,7 +30,7 @@ def get_version(session=None):
             return image_tag.lstrip("v")
         session = pyfluent.launch_fluent(mode="solver")
 
-    return session.get_fluent_version()
+    return session.get_fluent_version().value
 
 
 def get_version_for_file_name(version: Optional[str] = None, session=None):
```

Comment:
I'll look into if this and the previous function can some property of `FluentVersion` class (in a separate PR).